### ISSUE TITLE
Emerging Properties in Self-Supervised Vision Transformers

### ISSUE THREAD

**Issue by HYUNMIN-HWANG** [One-Day-One-Paper/One-Day-One-Paper#35]
### self-supervised ViT
- Mean Teacher self-distillation with no labels
- knowledge distillation : small network를 larger network의 결과를 모방하도록 훈련 시킨다.
- codistillation : student와 teacher 간에 같은 architecture를 갖고 있음, 훈련할 때 distillation을 사용함
- teacher 의 codistillation 은 student로 부터 또 distillation됨
- softmax along with cross entropy loss
![image](https://user-images.githubusercontent.com/70581043/149138152-965df825-8239-4720-833b-e54e72a6fb87.png)

### different views 
- global views : larger resolution, 224x224 image
- local views : smaller resolution, 96x96 image
- All crops are passed through the student while only the global views are passed through the teacher
- “local-to-global” :  training the student to interpolate context from a small crop

### Avoiding collapse
- collapse 문제 :  입력값과 상관없이 출력값이 하나의 차원이 dominate 하거나, 모든 차원에 대하여 uniform 한 상황을 말함
1. Centering
   - 한 차원이 dominant해지는 걸 막는다.
   - uniform distribution되는 걸 장려함
2. Sharpening
   - centering과 반대
   -  make the distribution more peaked
- 두 개를 balances 시킨다. -> batch에 덜 의존적이게 됨

### Results
- DINO has understood object semantics
![image](https://user-images.githubusercontent.com/70581043/149137704-60f6d2e1-1f6d-488b-886d-6bd2421f1605.png)
- ViT-S이 ResNet50과 거의 유사한 성능을 보임

---
- https://towardsdatascience.com/dino-emerging-properties-in-self-supervised-vision-transformers-summary-ab91df82cc3c
- https://deep-learning-study.tistory.com/827
